conf/spark-env.sh

```diff
@@ -8,6 +8,28 @@
 # CLI reads the "SPARK_HDFS_CONFIG_URL" marathon label in order to set
 # spark.mesos.uris
 
+parent_command=$(echo $@)
```

Comment:
let's use the changes from this PR https://github.com/mesosphere/spark-build/pull/526 when it's merged as they seem to work without issues.

Reply:
Since #526 was merged, rebased the branch on master to get these changes